packages/flutter/test/scheduler/ticker_test.dart

```diff
@@ -83,6 +83,54 @@ void main() {
     expect(ticker.toString(debugIncludeStack: true), contains('testFunction'));
   });
 
+  testWidgets('Ticker can be sped up with time dilation', (WidgetTester tester) async {
```

Comment:
Shouldn't there be tests for the timeDilationBehavior stuff?

Reply:
These are tests for the time dilation stuff :D